packages/javascript/bh-shared-ui/src/views/ZoneManagement/Details/MembersList.test.tsx

```
@@ -50,11 +50,11 @@ describe('MembersList', () => {
             <Routes>
                 <Route path={'/'} element={<MembersList selected='1' onClick={vi.fn()} />} />;
                 <Route
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)